oeps/oep-0026-arch-xapi-integration.rst

```diff
@@ -0,0 +1,483 @@
+==========================
+OEP-0026: xAPI Integration
+==========================
+
++-----------------+--------------------------------------------------------+
+| OEP             | :doc:`OEP-0026 </oeps/oep-0026-arch-xapi-integration>` |
++-----------------+--------------------------------------------------------+
+| Title           | xAPI Integration                                       |
++-----------------+--------------------------------------------------------+
+| Last Modified   | 2018-07-11                                             |
++-----------------+--------------------------------------------------------+
+| Authors         | Nimisha Asthagiri <nimisha@edx.org>                    |
++-----------------+--------------------------------------------------------+
+| Arbiter         | TODO                                                   |
++-----------------+--------------------------------------------------------+
+| Status          | Under Review                                           |
++-----------------+--------------------------------------------------------+
+| Type            | Architecture                                           |
++-----------------+--------------------------------------------------------+
+| Created         | 2018-07-11                                             |
++-----------------+--------------------------------------------------------+
+| `Review Period` | 2018-07-11 - 2018-08-28                                |
++-----------------+--------------------------------------------------------+
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Context
+-------
+
+Adaptive Learning
+=================
+
+As captured in the `Adaptive Learning Presentation`_, adaptive learning in Open edX 
+will enable the platform to respond to a learner's interactions in real-time, 
+automatically providing the learner with individualized support.  Data analysis 
+from earlier experimental attempts on edX.org and elsewhere show how adaptive 
+learning mechanisms can significantly improve the learner's efficiency, engagement, 
+and retention, while providing a more effective learning experience with metacognitive
+support for life-long learning.
+
+.. _Adaptive Learning Presentation: https://openedx.atlassian.net/wiki/spaces/AC/pages/641105978/Adaptive+Learning+Presentation
+
+Integration with Adaptive Engines
+=================================
+
+As an iterative path to this future, the immediate goal is to enable integrations with
+external adaptive engines.  In the far future, we may also implement our own open-sourced
+adaptive engines.  But in the meantime, we would like to establish standard APIs
+that (1) adaptive engines can use to receive real-time events from the Open edX LMS and
+(2) the Open edX LMS can use to query adaptive engines on what to present to learners.
+This document focuses on #1 with the need for scalable and real-time updates to adaptive
```

Comment:
@nasthagiri Another goal is to analyze the student's activity and build models based on the data we collected. It is better to train the models based on xAPI because then these data can be collected from multiple sources and not just the edx tracking events.

Reply:
Yes, good point.  I added this as a 3rd point.